patches/src/map_area.asm

```diff
@@ -495,10 +495,13 @@ draw_samus_indicator:
 .skip:
     rts
 
+org $82F9D4
```

Comment:
The address $82F9D4 is inside free space, so I'm not sure what this is trying to do, probably patching something that is already patched in this same file? In which case the patch should just be modified directly. As it is now, asar is failing to assemble `map_area.asm`. Also this `org $82F9D4` would cause the code below this (e.g. the `horizontal_scroll_hook` section) to also get relocated, which doesn't seem intended?

It might be better to address the map scrolling bug in a separate branch & PR since it doesn't seem related to the decompression?